packs/equipment-effects/effect-drowsy-sun-eye-drops.json

```diff
@@ -0,0 +1,42 @@
+{
+    "_id": "vX8SE5SXnspdzMYZ",
+    "img": "icons/consumables/potions/conical-ornate-purple.webp",
+    "name": "Effect: Drowsy Sun Eye Drops",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Drowsy Sun Eye Drops]</p>\n<p>You gain darkvision to a range of 30 feet.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 3
+        },
+        "publication": {
+            "license": "OGL",
+            "remaster": false,
+            "title": "Pathfinder Lost Omens: Travel Guide"
+        },
+        "rules": [
+            {
+                "key": "Sense",
+                "range": 30,
```

Comment:
Currently range is ignored for darkvision, but the structured data is included in case it ever does.

Reply:
:whyzo: